app/models/manageiq/providers/redhat/infra_manager.rb

```diff
@@ -205,9 +205,19 @@ def vm_migrate(vm, options = {})
       }
     }
 
+    started_time = Time.zone.now
     with_version4_vm_service(vm) do |service|
       service.migrate(migration_options)
     end
+
+    finished_event = nil
+    while finished_event.nil?
+      sleep timeout
+      finished_event = vm.ems_events.where(:event_type => %w(VM_MIGRATION_FAILED_FROM_TO VM_MIGRATION_DONE))
+                         .where(EventStream.arel_table[:timestamp].gt(started_time)).first
```

Comment:
Rubocop suggests `.find_by(EventStream.arel_table[:timestamp].gt(started_time))`
The rest looks good!